appveyor.yml

```diff
@@ -40,10 +40,8 @@ test_script:
 
     $success = $success -and $?
 
-    dotnet test .\test\WebJobs.Script.Tests.Integration\ -v q --no-build --filter "(FullyQualifiedName~Tests.NodeEndToEndTests|FullyQualifiedName~NodeContentTests|FullyQualifiedName~Tests.SecretsRepositoryTests|FullyQualifiedName~StandbyModeTests)"
+    dotnet test .\test\WebJobs.Script.Tests.Integration\ -v q --no-build

```

Comment:
So we've only actually been running a small subset of the tests on the CI? That explains why when I run locally I can never get a clean run in this branch, and I see lots of test hangs/failures. @fabiocav

Reply:
Yeah, and it does look like @brettsam hit your issues. Let's identify the failing/hanging tests and flag them for now.

Reply:
I now see this when running from the command-line. Some tests blow up and the run never continues. I'm going to track them down and skip them for now.

Reply:
Why skip them? I think we should just leave them failing and work to fix them. Having green CI runs when things are broken isn't really useful - false sense of confidence.

Reply:
We shouldn't block on getting all of the tests fixed to get some of the tests re-enabled. At least we're making progress and finding owners who can look at those tests one by one, instead of having to handle the entire set.

Reply:
The most important thing is to get parity between local runs and CI so it isn't a mystery when I'm getting failures locally but CI is green.

Reply:
Agreed. Explicitly flagging them (Skip) will do that.